examples/legalsimplifier/README.md

```diff
@@ -0,0 +1,65 @@
+# Legal Simplifier
+
+Legal Simplifier is an application that allows you to simplify legal documents - from terms and conditions of an insrance policy or a business contract. This example also shows how we can summarize contents of a large documents in chunks.
+
+Based on the document you upload, the size can vary and hence mgight find the need to split larger documents into chunks of 10,000 tokens to fit within GTP-4's context window.
```

Comment:
Imo this sentence can be removed, since the next section talks about how the document gets split up.